tests/unit/accuracy/test_math_grader.py

```
@@ -0,0 +1,385 @@
+    def test_last_number_fallback(self, grader: MathGrader) -> None:
+        answer, unparsed = grader._extract_with_flag("I think it might be 99 or 42")
+        assert answer == "42"
+        assert unparsed is True
+
+    def test_empty_response_marked_unparsed(self, grader: MathGrader) -> None:
+        answer, unparsed = grader._extract_with_flag("")
+        assert answer == ""
+        assert unparsed is True
+
+    def test_whitespace_only_response_marked_unparsed(self, grader: MathGrader) -> None:
+        answer, unparsed = grader._extract_with_flag("   \n\t  ")
```

Ruff (0.15.12)

[warning] Unpacked variable `answer` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)